web/backend/routes/root.ts

```diff
@@ -202,12 +200,36 @@ export default async function (fastify: FastifyInstance) {
     return api.getRuntimeServices(request.params.pid)
   })
 
-  typedFastify.get('/runtimes/:pid/logs', {
-    schema: {
-      params: { type: 'object', properties: { pid: { type: 'number' } }, required: ['pid'] }
-    }
-  }, async ({ params: { pid }, log }) =>
-    getLogsFromReadable(await api.getRuntimeAllLogsStream(pid), log))
+  typedFastify.get<{ Params: PidParam }>('/runtimes/:pid/logs/ws', {
+    schema: { params: pidParamSchema, hide: true },
+    websocket: true
```

Comment:
We can remove it once [this](https://github.com/fastify/fastify-swagger/pull/872) is released